doc/source/api.rst

```diff
@@ -74,55 +116,77 @@ ImageTable
     ImageTable
     ImageTable.as_dict
     ImageTable.has_decoded_images
+    ImageTable.load
+    ImageTable.from_table
 
+Natural Image Table
+===================
 
-Visualization Reference
------------------------
+.. currentmodule:: cvpy.image.NaturalImageTable
 
-.. currentmodule:: cvpy.visualization
+.. autosummary::
+    :toctree: generated/
+
+    NaturalImageTable
+    NaturalImageTable.as_dict
+    NaturalImageTable.has_decoded_images
+    NaturalImageTable.from_table
+    NaturalImageTable.load
+    NaturalImageTable.mask_image
```

Comment:
We expect users to call ImageTable.from_table() and ImageTable.load() functions. So we can remove the following.
NaturalImageTable.from_table
NaturalImageTable.load